styles/styles.css

```
@@ -31,37 +31,185 @@ body {
   font-family: PT Sans, sans-serif;
   font-size: 20px;
   line-height: 26px;
+  font-weight: bold;
   color: #000000;
   background-color: #FFFFFF;
+  text-align: center;
   text-decoration: none;
+}
+
+.navigation-link-user {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 21px;
+  text-align: center;
   text-transform: uppercase;
+  color: #FFFFFF;
+  text-decoration: none;
+  background-color: #756257;
+  border-radius: 4px;
 }
 
-.main-index {
-  background-image: url("../image/sedona.jpg");
-  background-color: #FFFFFF;
+.logo-img {
+  vertical-align: middle;
+}
+
+
+.welcome {
+  background-image: url("../image/background/sedona.jpg");
+  background-color: #83B3D3;
+  color: #FFFFFF;
   background-size: 100% auto;
   background-repeat: no-repeat;
 }
 
-.advantages-list {
+.slogan {
+  line-height: 36px;
+  font-weight: 700;
+  font-size: 30px;
   text-align: center;
+  color: #000000;
+  background-color: #FFFFFF;
+  text-transform: uppercase;
 }
 
-.mailing {
-  background-image: url("../image/background-masege.png");
+.text {
+  line-height: 36px;
+  font-weight: 400;
+  font-size: 22px;
+  text-align: center;
+  color: #000000;
+  background-color: #FFFFFF;
+}
+
+.advantages-title {
+  line-height: 28px;
+  font-size: 24px;
+  text-align: center;
+  font-weight: bold;
+  color: #000000;
+  text-transform: uppercase;
+}
+
+.advantages-priority {
+  background-color: #83B3D3;
+  text-align: center;
+}
+
+.advantages-description {
+  line-height: 21px;
+  font-size: 18px;
+  color: #000000;
+  text-align: center;
+}
+
+.features-item {
+  background-color: #rgba(131, 179, 211, 0, 12);
```

Через решетку пишутся цвета в формате HEX, когда мы используем rgba, решетка уже не нужна:
```suggestion
  background-color: rgba(131, 179, 211, 0, 12);
```